tests/validator_test.go

```diff
@@ -320,13 +324,415 @@ var _ = Describe("Template validator", func() {
 				progressing := conditionsv1.FindStatusCondition(obj.Status.Conditions, conditionsv1.ConditionProgressing)
 
 				return obj.Status.Phase == lifecycleapi.PhaseDeploying &&
-					available.Status == core.ConditionTrue &&
-					progressing.Status == core.ConditionTrue
+					available.Status == k8sv1.ConditionTrue &&
+					progressing.Status == k8sv1.ConditionTrue
 			}, timeout)
 			Expect(err).ToNot(HaveOccurred(), "SSP should be available, but progressing.")
 
 			err = WatchChangesUntil(watch, isStatusDeployed, timeout)
 			Expect(err).ToNot(HaveOccurred(), "SSP status should be deployed.")
 		})
 	})
+
+	FContext("VM should", func() {
+		var (
+			vm  *v1.VirtualMachine
+			vmi *v1.VirtualMachineInstance
+			templateWithRules, templateWithRulesOptional, templateWithIncorrectRules,
+			templateWithIncorrectRuleJustWarning, templateWithoutRules templatev1.Template
+		)
+
+		BeforeEach(func() {
+			templateWithRules = test_utils.TemplateWithRules()
+			templateWithRulesOptional = test_utils.TemplateWithRulesOptional()
+			templateWithIncorrectRules = test_utils.TemplateWithIncorrectRules()
+			templateWithIncorrectRuleJustWarning = test_utils.TemplateWithIncorrectRulesJustWarning()
+			templateWithoutRules = test_utils.TemplateWithoutRules()
+
+			for _, template := range []templatev1.Template{templateWithRules, templateWithRulesOptional,
+				templateWithIncorrectRules, templateWithIncorrectRuleJustWarning, templateWithoutRules} {
+				template.Namespace = strategy.GetNamespace()
+				Expect(apiClient.Create(ctx, &template)).ToNot(HaveOccurred(), "Failed to create template: %s", template.Name)
+				Expect(apiClient.Get(ctx, client.ObjectKey{Name: template.Name, Namespace: template.Namespace},
+					&template)).ToNot(HaveOccurred())
+			}
+			vmi = test_utils.NewRandomVMIWithBridgeInterface(strategy.GetNamespace())
+		})
+
+		It("be created without template", func() {
+			vm = test_utils.NewRandomVirtualMachine(vmi, false)
+			Expect(apiClient.Create(ctx, vm)).ToNot(HaveOccurred(), "Failed to create VM")
+		})
+		It("be created from template without rules", func() {
+			vm = test_utils.NewRandomVirtualMachine(vmi, false)
+			vm.ObjectMeta.Annotations = map[string]string{
+				"vm.kubevirt.io/template":           "fedora-desktop-small",
+				"vm.kubevirt.io/template-namespace": strategy.GetTemplatesNamespace(),
+			}
+			Expect(apiClient.Create(ctx, vm)).ToNot(HaveOccurred(), "Failed to create VM")
+		})
+		It("[test_id:5033]: Template with validations, VM without validations", func() {
+			vmi.Spec.Domain.CPU = &v1.CPU{
+				Cores: 2,
+			}
+			vmi.Spec.Domain.Machine.Type = "q35"
+			vmi.Spec.Domain.Resources.Requests = k8sv1.ResourceList{
+				k8sv1.ResourceMemory: resource.MustParse("128M"),
+			}
```

Comment:
These values are always assigned to create a VM that needs to pass validation. It would be clearer to extract them to a function with descriptive name like `CreateVmiToPassValidation`, and use is in every test  that needs it.